docs/build/install.md

```diff
@@ -62,33 +63,33 @@ subql-node <command>
 ```
 
 ::: info Note
-If you are using Docker or hosting your project in SubQuery Projects, you do can skip this step. This is because the SubQuery node is already provided in the Docker container and the hosting infrastructure. 
+If you are using Docker or hosting your project in SubQuery Projects, you do can skip this step. This is because the SubQuery node is already provided in the Docker container and the hosting infrastructure.
```

Comment:
```suggestion
If you are using Docker or hosting your project in SubQuery Projects, you can skip this step. This is because the SubQuery node is already provided in the Docker container and the hosting infrastructure.
```